app/models/Campaign.js

```diff
@@ -34,8 +34,12 @@ const campaignSchema = new mongoose.Schema({
   msg_menu_signedup_external: String,
   msg_menu_signedup_gambit: String,
   msg_no_photo_sent: String,
-  msg_relative_reminder_signup: String,
-  msg_relative_reminder_reportback: String,
+
+  // Exposed messages.
+  messages: {
```

Comment:
Also, should this be `signup_reminder`? Im just worried `signup` could be confused with a "Thanks for signing up" or something similar